Bender.yml

```diff
@@ -0,0 +1,20 @@
+package: 
+  name: udma_external_per
+  authors: 
+    - "Pasquale Davide Schiavone <pschiavo@iis.ee.ethz.ch>"
+
+dependencies:
+  udma_core: { git: "git@github.com:pulp-platform/udma_core.git", rev: "82300ee1aae9faf6384d35ff6d6ca6ecf7f1db84" }
```

Comment:
One question, though: Would it not be worth tagging a new version and using that here? `version` is preferred over `rev`

Reply:
I agree, would add, although I cannot create releases. @adimauro-iis would it be possible to create a v1.0.1 or v1.1.0 release of https://github.com/pulp-platform/udma_core with the newest commit adding the Bender.yml file?

Reply:
There is a [`v1.0.1` tag for `udma_core`](https://github.com/pulp-platform/udma_core/tree/v1.0.1). You may want to add this in a new PR.